http/daemon/server.go

```diff
@@ -144,13 +144,15 @@ func (s HTTPServer) ListServicesWithOptions(w http.ResponseWriter, r *http.Reque
 	var opts v11.ListServicesOptions
 	opts.Namespace = r.URL.Query().Get("namespace")
 	services := r.URL.Query().Get("services")
-	for _, svc := range strings.Split(services, ",") {
-		id, err := flux.ParseResourceID(svc)
-		if err != nil {
-			transport.WriteError(w, r, http.StatusBadRequest, errors.Wrapf(err, "parsing service spec %q", svc))
-			return
+	if services != "" {
```

Comment:
Doesn't the new code behave exactly the same as before, given that `strings.Split("", ",")` is `[]`, and hence the for-loop is a no-op in that case?

Reply:
`strings.Split("", ",")` is not `[]`.

Reply:
Ah, very true. I was misled by the default formatter.

> 	fmt.Println([]string{""})
> []

Well done, golang, well done.